### PR TITLE
Add metrics implementation

### PR DESCRIPTION
**Pull request by travisgroth**
Establish initial implementation of #35 

Subsequent PRs for authenticate, authorize, proxy client side, and misc values.

**Checklist**:
- [x] updated docs
- [x] unit tests added
- [x] related issues referenced
- [x] updated CHANGELOG.md
- [x] ready for review

### CODE REVIEW

**Comment by desimone** [comment]
About to jump on a plane but I plane and will give this PR a spin.

File: internal/config/options.go [hunk 1]
```diff
@@ -162,6 +165,7 @@ func NewOptions() Options {
 		IdleTimeout:       5 * time.Minute,
 		RefreshCooldown:   time.Duration(5 * time.Minute),
 		AllowWebsockets:   false,
+		MetricsAddr:       "127.0.0.1:9090",
```
[the remark this answers is not in the document]
Comment (travisgroth) [reply internal/config/options.go]: I could argue either way.  If you're thinking we default off, that works for me.

**Comment by desimone** [comment]
Yeah let’s default off. Thanks! Worked great when I brought it down.

**Comment by desimone** [comment]
🍾

**Pull request merged**